plugins/git/service/git.thrift

```diff
@@ -209,5 +209,11 @@ service GitService
     1:string repoId_,
     2:string hexOid_,
     3:string path_,
-    4:string localModificationsFileId_)
+    4:string localModificationsFileId_),
+
+   /**
+   * Check whether there is at least one repository
+   * in the workspace directory.
+   */
+    bool isEnabled()
```

Comment:
I think `enabled` is too generic and bad for what this function tries to convey. We can selectively disable plugins at some places, some plugins (such as cpp-reparse) have a manual "disable" flag that even if the plugin *is loaded* it should not serve the clients, etc. Let's turn this into a more descriptive name of what the API is trying to convey.

Reply:
I renamed it to `isRepositoryAvailable`.

Reply:
Thanks! Much more clearer this way.